app/bootstrap/errors.php

```diff
@@ -10,7 +10,8 @@
                 array(
                     $container['config']['error-email']),
                 "404 Error on " . $container['config']['site-name'],
-                "<pre>" . $exception . "</pre>"
+                "<pre><b>Route: </b>" . $request->getUri()->getPath() . "</pre>" . 
```

Comment:
__[PHP_CodeSniffer]__ <sub>posted by Sider</sub>

Whitespace found at end of line